app/src/main/res/raw/data.json

```diff
@@ -1738,5 +1738,2365 @@
+	[
+    {"EN":{
+        "word": "spring",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "весна",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "autumn",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "лето",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "summer",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "лето",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "autumn",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "осень",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "fall",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "осень",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "holiday",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "отпуск",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "accommodation",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "проживание",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "book",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "забронировать",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "airplane",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "самолет",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "train",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "поезд",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "bus",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "автобус",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "ship",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "корабль",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "mountain",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "гора",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "sea",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "море",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "ocean",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "океан",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "hotel",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "отель",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "borders",
```

Comment:
probably better in single form

Reply:
I think they use plural form more often.